src/indicator/algorithm/calculator/elderRay.js

```diff
@@ -0,0 +1,73 @@
+import d3 from 'd3';
+import exponentialMovingAverage from './exponentialMovingAverage';
+import {identity} from '../../../util/fn';
+import {rebind} from '../../../util/rebind';
+
+export default function() {
+
+    var value = identity;
+
+    var highValue = function(d, i) { return d.high; },
+        lowValue = function(d, i) { return d.low; };
+
+    var ema = exponentialMovingAverage()
+        .windowSize(13);
+
+    var elderRay = function(data) {
+
+        ema.value(value);
+        var x = ema(data);
+
+        var elderRay = d3.zip(data, x)
+            .map(function(d) {
```

Comment:
It also means that for indicator renderers, where you do `root(d)` you can always assume that this will return an object rather than undefined. I think all of this is captured here:

https://github.com/ScottLogic/d3fc/blob/master/src/indicator/design-guidelines.md